docs/content/learn/tutorials/control/basic-profile.md

```diff
@@ -0,0 +1,1282 @@
+
+## Creating an OSCAL Profile
+
+The examples below illustrate the top-level structure of the OSCAL profile model.
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+```xml {linenos=table}
+<?xml version="1.0" encoding="UTF-8"?>
+<profile xmlns="http://csrc.nist.gov/ns/oscal/1.0"
+    uuid="9510e179-7744-4afa-a9d3-92beadffd85f">
+    <!-- To be filled in: -->
+    <metadata/>
+    <import/>
+    <merge/>
+    <modify/>
+    <back-matter/>
+</profile>
+```
+
+The root of the OSCAL profile model is [`<profile>`](/reference/latest/profile/xml-reference/#/profile).
+
+In the example above, the contents of the `<profile>` element is provided as empty data items. These are included to illustrate the content model of an OSCAL profile, and we will be covering each element's syntax later in this tutorial.
+
+The `@id` attribute (on line 3) is the document's *universally unique identifier* (UUID), a unique 128-bit number displayed as a string of hyphenated hexadecimal digits as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122). OSCAL documents use a version 4 UUID (randomly generated) to uniquely identify the document.
+
+A `<profile>` contains:
+- `<metadata>` (required) - Provides document metadata for the profile. As OSCAL Metadata sections use a shared structure across all models, refer to the [metadata tutorial](metadata-tutorial) for more information.
+- `<import>` (required) - This is explored below in the [import phase](#import-phase) section of this tutorial.
+- `<merge>` (optional) - This is explored below in the [merge phase](#merge-phase) section of this tutorial.
+- `<modify>` (optional) - This is explored below in the [modify phase](#modify-phase) section of this tutorial.
+- `<back-matter>` (optional) – Contains references used within the profile. Use of `<back-matter>` is not covered in this tutorial.
+{{% /tab %}}
+{{% tab %}}
+```json {linenos=table}
+{
+  "profile": {
+    "uuid": "9510e179-7744-4afa-a9d3-92beadffd85f",
+    // To be filled in:
+    "metadata": {},
+    "import": {},
+    "merge": {},
+    "modify": {},
+    "back-matter": {}
+  }
+}
+```
+
+The root of the OSCAL profile model is [`profile`](/reference/latest/profile/xml-reference/#/profile).
+
+In the example above, the contents of the `profile` object are provided as empty data items. These are included to illustrate the content model of an OSCAL profile, and we will be covering each element's syntax later in this tutorial.
+
+The `id` property (on line 3) is the document's *universally unique identifier* (UUID), a unique 128-bit number displayed as a string of hyphenated hexadecimal digits as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122). OSCAL documents use a version 4 UUID (randomly generated) to uniquely identify the document.
```

Comment:
Ditto here, was there an intent to calling this the `id` property that I misunderstand?

```suggestion
The `uuid` property (on line 3) is the document's *universally unique identifier* (UUID), a unique 128-bit number displayed as a string of hyphenated hexadecimal digits as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122). OSCAL documents use a version 4 UUID (randomly generated) to uniquely identify the document.
```